app/Console/Commands/UpdateDurHasDatasetVersionsGat7257.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Models\DurHasDatasetVersion;
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+
+class UpdateDurHasDatasetVersionsGat7257 extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'app:update-dur-has-dataset-versions-gat7257';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Command for task GAT-7257: Multiple entries for same link in dur_has_dataset_versions table';
+
+    /**
+     * Execute the console command.
+     */
+    public function handle()
+    {
+        Schema::disableForeignKeyConstraints();
```

Comment:
if OK to do all this..

https://laravel.com/docs/12.x/database#database-transactions

wrap in DB transaction if something goes wrong and everything is lost?

Reply:
I understand, but it is not a must.